tracer/src/Datadog.Trace.Tools.Runner/LegacyCommand.cs

```diff
@@ -40,7 +40,8 @@ private void Execute(InvocationContext context)
                 context,
                 ApplicationContext.RunnerFolder,
                 ApplicationContext.Platform,
-                _legacySettings);
+                _legacySettings,
+                false);
```

Comment:
```suggestion
                reducePathLength: false);
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4558/commits/eb51dd2e7046f8d90ae8e43a368ce9587b55f5d4